eng/scripts/Swagger-APIView.ps1

```diff
@@ -0,0 +1,191 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$TempDirectory,
+    [Parameter(Mandatory = $true)]
+    [string]$ArtiFactsDirectory
+)
+
+. "$PSScriptRoot/ChangedFiles-Functions.ps1"
+. "$PSScriptRoot/Logging-Functions.ps1"
+
+<#
+.DESCRIPTION
+  Get the readme.md file associated with a swagger file.
+
+.PARAMETER SwaggerFile
+  Path to a swagger files inside the 'specification' directory.
+
+.OUTPUTS
+  the readme.md file associated with the swagger file or null if not found.
+#>
+function Get-SwaggerReadMeFile {
```

Comment:
How about moving this function and the subsequent one into a common PS script? They can be reused by other CI checks as well.

Reply:
looks like this could also happen to `Get-ResourceProviderFromReadMePath`